examples/cms-sitecore/next.config.js

```diff
@@ -0,0 +1,7 @@
+const url = process.env.SITECORE_CONTENTHUB_PUBLIC_URL;
+const { hostname } = new URL(url);
+module.exports = {
+  images: {
+    domains: [ hostname , 'dummyimage.com'],
```

Comment:
Does this need updated?

Reply:
No, I am using dummyimage as a default fallback, in case no image returned.